Cargo.toml

```diff
@@ -27,6 +27,7 @@ actix-web = "0.7"
 simplelog = "0.8"
 base64 = "0.12"
 percent-encoding = "2.1"
+port_check = "0.1"
```

Comment:
We now have this listed twice. Can you just remove it from the dev deps below?

Reply:
Oh, definitely! Done.